conf/deployment/aws/gp2_csi_storage_class.yaml

```diff
@@ -0,0 +1,4 @@
+---
+# This config is for AWS deployment with gp2 CSI storage class provisioning
+DEPLOYMENT:
+  customized_deployment_storage_class: 'gp2-csi'
```

Comment:
please add `customized_deployment_storage_class` in conf/README.md under `DEPLOYMENT`

Reply:
Yes, that will do.